app/views/homes/show.html.erb

```diff
@@ -72,5 +72,12 @@
     </table>
   </p>
 
+  <!--
+    /// DO NOT REMOVE ///
+    /// This counter is from http://www.webestools.com/visitors-counter-free-number-unique-visitors-users-visits-statistics.html ///
+    /// DO NOT REMOVE ///
+  -->
+  <div style="text-align:center;"><script type="text/javascript" src="http://services.webestools.com/cpt_visitors/40122-13-5.js"></script></div><a href="http://www.webestools.com/" style="display:block;text-align:center;" title="Tools services webmasters counters generators scripts tutorials free"></a>
```

Comment:
Maybe instead of the comment, put a class on the div like `visitor-counter` to give future devs an idea of what it is? It even has documentation built in linking back to the site.